src/Websites/Websites/Cmdlets/Certificates/NewAzureWebAppCertificate.cs

```diff
@@ -94,7 +94,7 @@ public override void ExecuteCmdlet()
                     {
                         //Default certName is HostName
                         certName = Name != null ? Name : HostName;
-                        createdCertdetails = (PSCertificate)WebsitesClient.CreateCertificate(ResourceGroupName, certName, certificate);
+                        createdCertdetails = WebsitesClient.CreateCertificate(ResourceGroupName, certName, certificate);
```

Comment:
SDK types should not be exposed to the user, because they can break any time.
We should wrap the SDK type with a custom type (PSCertificate in this case), so that when breaking change happens to SDK, we can work-around it in the wrapper class

Reply:
Updated as suggested.